schema/schema.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS Calendar (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	user_fk INT NULL,
+	name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS Meeting (
+	id INT AUTO_INCREMENT PRIMARY KEY,
+	name VARCHAR(50) NOT NULL,
+	startDateTime datetime NOT NULL,
+	endDateTime datetime NOT NULL,
+	createdAt datetime default CURRENT_TIMESTAMP NOT NULL,
+	updatedAt datetime default CURRENT_TIMESTAMP NOT NULL on update CURRENT_TIMESTAMP,
+	user_calendar_fk INT NULL,
+	organizing_event INT NULL,
```

Comment:
Maybe both user_calendar_fk and organizing_event should be NOT NULL

Reply:
let's leave organizing_event as nullable for now else every meeting, even ones that are organizing events, will need to have a non-Null FK value there.

I agree that user_calendar_fk should be non-null though, so I will update.